pkg/policy/l4.go

```diff
@@ -463,7 +501,36 @@ func NewL4Policy(revision uint64) *L4Policy {
 		Ingress:  L4PolicyMap{},
 		Egress:   L4PolicyMap{},
 		Revision: revision,
+		users:    make(map[*EndpointPolicy]struct{}),
+	}
+}
+
+// insertUser adds a user to the L4Policy so that incremental
+// updates of the L4Policy may be fowarded to the users of it.
+func (l4 *L4Policy) insertUser(user *EndpointPolicy) {
+	l4.mutex.Lock()
+	if l4.users == nil {
+		log.Warningf("L4Filter: Skipping setting user on a stale policy")
```

Comment:
This seems to be a Warning message, what can the user do if it seems this message?

Reply:
Good question :-) Changed to Debug.